### ISSUE TITLE
istio.io setup/install/multicluster/multi-primary_multi-network/test.sh test fails on `main` branch

### ISSUE THREAD

**Issue by ericvn** [istio/istio#29388]
(**NOTE**: This is used to report product bugs:
  To report a security vulnerability, please visit <https://istio.io/about/security-vulnerabilities>
  To ask questions about how to use Istio, please visit <https://discuss.istio.io>)

**Bug description**

Trying to move the istio.io tests from the `release-1.8` branch to `main` and the test now fails. The tail end of the log (https://storage.googleapis.com/istio-prow/pr-logs/pull/istio_istio.io/8562/doc.test.multicluster_istio.io/1334602913783222272/artifacts/tests-setup-multicluster-42277d/TestDocs/setup/install/multicluster/multi-primary_multi-network/test.sh/test.sh/_test_context/test.sh_output.txt):
```
✔ Ingress gateways installed
- Pruning removed resources
✔ Installation completeWaiting for the east-west gateway to have an external IP
VERIFY FAILED snip_install_the_eastwest_gateway_in_cluster1_2 (timeout after 120s):
received:
"Error from server (NotFound): services "istio-eastwestgateway" not found"
expected:
"NAME                    TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)   AGE
istio-eastwestgateway   LoadBalancer   10.80.6.124   34.75.71.237   ...       51s"

real	2m22.422s
user	0m16.059s
sys	0m8.119s
VERIFY FAILED snip_install_the_eastwest_gateway_in_cluster2_2 (timeout after 120s):
received:
"Error from server (NotFound): services "istio-eastwestgateway" not found"
expected:
"NAME                    TYPE           CLUSTER-IP    EXTERNAL-IP    PORT(S)   AGE
istio-eastwestgateway   LoadBalancer   10.0.12.121   34.122.91.98   ...       51s"

...
...
...
deployment.apps/sleep created
Waiting for deployment "helloworld-v1" rollout to finish: 0 of 1 updated replicas are available...
deployment "helloworld-v1" successfully rolled out
deployment "sleep" successfully rolled out
Waiting for deployment "helloworld-v2" rollout to finish: 0 of 1 updated replicas are available...
deployment "helloworld-v2" successfully rolled out
Waiting for deployment "sleep" rollout to finish: 0 of 1 updated replicas are available...
deployment "sleep" successfully rolled out
Verifying helloworld v1 deployment
Verifying helloworld v2 deployment
Verifying sleep deployment in kind-doc-cluster1
Verifying sleep deployment in kind-doc-cluster3
Verifying load balancing from kind-doc-cluster1
VERIFY FAILED snip_verifying_crosscluster_traffic_1 (timeout after 120s):
received:
"  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:01 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:02 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:03 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:04 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:05 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:06 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:07 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:08 --:--:--     0
  0     0    0     0    0     0      0      0 --:--:--  0:00:09 --:--:--     0upstream connect error or disconnect/reset before headers. reset reason: local reset, transport failure reason: TLS error: 268435612:SSL routines:OPENSSL_internal:HTTP_REQUEST
100   175  100   175    0     0     17      0  0:00:10  0:00:10 --:--:--    36
100   175  100   175    0     0     17      0  0:00:10  0:00:10 --:--:--    46"
expected:
"Hello version: v2, instance:"
```

I don't know if the docs need updating or if there is an issue with the underlying code. If the later, then there would be a new integration test created such that this problem was found earlier and not rely on the istio.io tests to find the issue.

[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Extensions and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure
[ ] Upgrade

**Expected behavior**

Istio.io tests should work.

**Steps to reproduce the bug**

Run the istio.io test or try and follow the instructions on the content/en/docs/setup/install/multicluster/multi-primary_multi-network/index.md page.

**Version** (include the output of `istioctl version --remote` and `kubectl version --short` and `helm version --short` if you used Helm)

`main` branch

**How was Istio installed?**

by test framework

**Environment where the bug was observed (cloud vendor, OS, etc)**

Istio pipeline

Additionally, please consider running `istioctl bug-report` and attach the generated cluster-state tarball to this issue.
Refer [cluster state archive](http://istio.io/help/bugs/#generating-a-cluster-state-archive) for more details.

**Comment by linsun** [comment]
@nmittler @stevenctl could one of you look at this?  This is preventing us from testing istio.io on master.

**Comment by nmittler** [comment]
@ericvn do you have links to the prow jobs? I don't seem to be able to navigate to other folders with the storage URL

**Comment by ericvn** [comment]
@nmittler You can look at the details links in the istio.io PR: https://github.com/istio/istio.io/pull/8562 for a more recent run then the link I posted above.

**Comment by nmittler** [comment]
@ericvn this should be fixed now with the latest master/1.9 branches. Closing for now. Go ahead and re-open if it's still an issue.
